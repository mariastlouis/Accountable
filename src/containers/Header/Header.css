@import url("https://fonts.googleapis.com/css?family=Pathway+Gothic+One");
@import url("https://fonts.googleapis.com/css?family=Cabin");
@import url("https://fonts.googleapis.com/css?family=Open+Sans:300,400,600");
@import url("https://fonts.googleapis.com/css?family=Raleway:300,400,500,600");
body {
  font-family: "Open Sans", sans-serif;
  color: #545252; }

h1, h2, h3 {
  font-family: "Raleway", sans-serif;
  -webkit-margin-before: 0;
  -webkit-margin-after: 0;
  -webkit-margin-start: 0px;
  -webkit-margin-end: 0px; }

h1 {
  font-size: 40px;
  font-weight: 400; }

h2 {
  font-weight: 300;
  display: block; }

p {
  font-size: 16px;
  font-family: "Open Sans", sans-serif;
  font-weight: 400; }

header {
  display: flex;
  flex-direction: row;
  justify-content: space-between; }

.nav-bar {
  margin-top: 30px;
  display: -ms-flexbox;
  display: flex;
  -ms-flex-direction: row;
  flex-direction: row;
  justify-content: center;
  align-items: center; }

.nav {
  text-decoration: none;
  font-size: 25px;
  font-weight: 300;
  color: #3a6985;
  padding: 10px; }

.nav:hover {
  background: #941b20;
  color: white; }
