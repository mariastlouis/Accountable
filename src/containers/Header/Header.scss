
@import '../../assets/variable.scss';

header {
  display: flex;
  flex-direction: row;
  justify-content: space-between;
}

.nav-bar {
  margin-top: 30px;
  display: -ms-flexbox;
  display: flex;
  -ms-flex-direction: row;
  flex-direction: row;
  justify-content: center;
  align-items: center;
}
.nav {
  text-decoration: none;
  font-size: 25px;
  font-weight: 300;
  color: $accent-blue;
  padding: 10px;
}

.nav:hover{
  background: $accent-red;
  color: white;
}

