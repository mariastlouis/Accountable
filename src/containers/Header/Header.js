import React from 'react';
import accountableLogo from '../../assets/images/accountable-logo2-02.png';
import { Link} from 'react-router-dom';
import './Header.css';

const Header = () => {
  return (
    <div className = 'header'>
      <header>
        <div className = 'logo'>
          <Link to = {'/'}>
            <img src = {accountableLogo} alt = 'logo' />
          </Link>
        </div>
      </header>
    </div>  
  );
};

export default Header;

// <div className = 'nav-bar'>
// <NavLink to = '/lawmakers' className = 'nav'>LAWMAKERS </NavLink>
// <NavLink to = '/bills' className = 'nav'>BILLS </NavLink>
// <NavLink to = '/committees' className = 'nav'>COMMITTEES </NavLink>
// </div>