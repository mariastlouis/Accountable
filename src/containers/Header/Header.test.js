import Header from './Header.js';
import React from 'react';
import { shallow } from 'enzyme';

describe('header comoponent tests', () => {
  let renderedComponent;

  beforeEach(() => {
    renderedComponent = shallow(<Header />);
  });

  it('should render', () =>{
    expect(renderedComponent).toBeDefined();
  });

  it('should match the snapshot', () => {
    expect(renderedComponent).toMatchSnapshot();
  });

});