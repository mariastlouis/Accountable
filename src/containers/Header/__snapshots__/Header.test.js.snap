// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`header comoponent tests should match the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Header />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <header>
        <div
                className="logo"
        >
                <Link
                        replace={false}
                        to="/"
                >
                        <img
                                alt="logo"
                                src="accountable-logo2-02.png"
                        />
                </Link>
        </div>
</header>,
      "className": "header",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="logo"
>
          <Link
                    replace={false}
                    to="/"
          >
                    <img
                              alt="logo"
                              src="accountable-logo2-02.png"
                    />
          </Link>
</div>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <Link
            replace={false}
            to="/"
>
            <img
                        alt="logo"
                        src="accountable-logo2-02.png"
            />
</Link>,
          "className": "logo",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": <img
              alt="logo"
              src="accountable-logo2-02.png"
/>,
            "replace": false,
            "to": "/",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "alt": "logo",
              "src": "accountable-logo2-02.png",
            },
            "ref": null,
            "rendered": null,
            "type": "img",
          },
          "type": [Function],
        },
        "type": "div",
      },
      "type": "header",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <header>
          <div
                    className="logo"
          >
                    <Link
                              replace={false}
                              to="/"
                    >
                              <img
                                        alt="logo"
                                        src="accountable-logo2-02.png"
                              />
                    </Link>
          </div>
</header>,
        "className": "header",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <div
            className="logo"
>
            <Link
                        replace={false}
                        to="/"
            >
                        <img
                                    alt="logo"
                                    src="accountable-logo2-02.png"
                        />
            </Link>
</div>,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <Link
              replace={false}
              to="/"
>
              <img
                            alt="logo"
                            src="accountable-logo2-02.png"
              />
</Link>,
            "className": "logo",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "children": <img
                alt="logo"
                src="accountable-logo2-02.png"
/>,
              "replace": false,
              "to": "/",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "alt": "logo",
                "src": "accountable-logo2-02.png",
              },
              "ref": null,
              "rendered": null,
              "type": "img",
            },
            "type": [Function],
          },
          "type": "div",
        },
        "type": "header",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
